geps/overview.md

```diff
@@ -71,7 +71,8 @@ proposed changes in our API. In some cases, these changes will be documentation
 only, but in most cases, some API changes will also be required. It is important
 that every new feature of the API is marked as "Experimental" when it is
 introduced. Within the API, we use `<gateway:experimental>` tags to denote
-experimental fields.
+experimental fields. Within Golang packages (conformance tests, CLIs, e.t.c.) we
```

Comment:
The original justification for the build flag was provisional status of the GEP. Is the flag still necessary now that it's experimental? @shaneutt

Reply:
Because it's a replacement for the normal CLI and test suite, the intention was to leave it, however I suppose we can have a discussion about removing the build tag if that's helpful to people. :thinking:

Reply:
The build tag is a (soft) blocker for our adoption personally

Reply:
Would also be in favor of dropping experimental build tag here, ideally pre-0.8.0 so it's a bit easier to report results.

Reply:
Sounds good https://github.com/kubernetes-sigs/gateway-api/issues/2242